### PR TITLE
initial nttw4

### PR DESCRIPTION
**Pull request by dericed**
ping @JeromeMartinez @ablwr

### CODE REVIEW

**Review by ablwr — APPROVED** [review]
Looks ok to me but didn't test locally

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+<h2>Where</h2>
+<p><a href="http://www.osaarchivum.org/">Vera and Donald Blinken Open Society Archives</a>, Arany János u. 32, 1051 Budapest, Hungary</p>
+
+<p>
+{% image '@NoTimeToWaitBundle/Resources/public/img/OSAatCEU_BudapestHungary_02.jpg' %}
+  <img src="{{ asset_url }}" alt="Open Society Archives at CEU in Budapest, Hungary" class="img-responsive center-xs" title="Open Society Archives at CEU in Budapest, Hungary">
+{% endimage %}
+<br />
+<em>Photo Credit: <a href="">Vera and Donald Blinken Open Society Archives</a></em></p>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig]: Bad copy/paste, I changed the credit name (please confirm it).
Do you have an URL for the credit?

File: src/NoTimeToWaitBundle/Resources/views/Default/index.html.twig [hunk 2]
```diff
@@ -20,7 +20,7 @@
           No Time To Wait! Symposium originally was formed as part of the PREFORMA project as a "3 day symposium of workshops &amp; presentations on the standardization &amp; use of FFV1 &amp; Matroska in archives" held before an IETF event in Berlin and hosted by Deutsche Kinemathek, Zuse Institute Berlin and MediaArea.net. The following year, the conference continued in Vienna, hosted by Austrian Film Museum and MediaArea.net with support of AV-RD, with a broader scope defined as "A symposium focused on open media, open standards, and digital audiovisual preservation." and "A 2 day symposium on the intersection of open media, standardization, and audiovisual preservation." The Year 2 theme was "Open Media, Open Formats, Open Archives".
         </p>
         <p>
-          Both conferences used the tagline of:
+          Conferences used the tagline of:
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/index.html.twig:23]: I removed `both`, @dericed other suggestion?

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2695.352088617675!2d19.051191951583835!3d47.502533979075764!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4741dc14ca087bed%3A0x903b5ebe0d68ad4f!2sOSA+Archives!5e0!3m2!1sen!2sus!4v1549293616291" width="500" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+
+<p>Parts of the workshop will also be accessible remotely. More info soon.</p>
+
+<h2>Program</h2>
```
Comment (JeromeMartinez) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig:55]: Bad copy/paste, I changed it.

**Review by JeromeMartinez — CHANGES_REQUESTED** [review]
I added the technical boilerplate.
+ some copy/paste fixes
But 1 missing info.

Please review my changes then squash in 1 commit.

**Review by JeromeMartinez — APPROVED** [review]

**Pull request merged**